pkg/osdCloud/aws.go

```diff
@@ -125,9 +127,19 @@ func GenerateJumpRoleCredentials(client aws.Client, awsAccountID, region, sessio
 		return nil, err
 	}
 
+	jumpRoleKey := ProdJumproleConfigKey
```

Comment:
Can we also add a message here noting that using `osdctl` to assume support role credentials will be deprecated in a future version in favor of backplane?

I don't want to obliterate anyone's workflow, but at the same time we should probably start the deprecation process sooner rather than later.

Reply:
(It doesn't have to be _here_ specifically, but inside this function somewhere, or somewhere it makes sense that we're _not_ using the `-i` flag.)

Reply:
In https://github.com/openshift/osdctl/pull/259 I made changes to the workflow for access AWS accounts for -C that no longer require hive access, so I don't think we should deprecate -C. I've refactored now a number of commands, like osdctl health, to use this flow as it needs the same functionality as -C anyway.

Reply:
Hmm, okay.  I thought the reason for the deprecation was mostly so that we only have the backplane access available, but it does make sense to have a backup method in case backplane itself goes down.

Reply:
Part of the deprecation discussion (at least from my perspective) was the hive access requirement was annoying and backplane was much simpler. Now the osdctl method is also simple and is only dependent on aws credentials and ocm credentials, so I changed my stance on it. For now, I would prefer to keep it just for backwards compatibility and ease of use.